pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/util/SendGridMailUtil.java

```diff
@@ -96,21 +90,4 @@ private static void submitMailMessage(Mail mailMessage, String sendGridApiKey, S
             log.info("Successfully submitted message to SendGrid with subject {}", mailMessage.subject);
         }
     }
-
-    private static SendGrid createSendGridInstance(String sendGridApiKey, String proxy) {
```

Comment:
Moved to [SendGridFactory](https://github.com/broadinstitute/ddp-study-server/pull/2683/files#r1335942101).